src/stan/mcmc/chains.hpp

```diff
@@ -292,6 +291,91 @@ namespace stan {
         return ess;
       }
 
+      /**
+       * Returns the effective sample size for the specified parameter
+       * across all kept samples.
+       *
+       * The implementation is close to the effective sample size
+       * description in BDA3 (p. 286-287).  See more details in Stan
+       * reference manual section "Effective Sample Size".
+       *
+       * Current implementation takes the minimum number of samples
```

Comment:
There's no minimums taken here.  It all just uses the cross-chain mean and variance estimates to adjust within chain estimates.  But I wouldn't even try to say that, I'd just point to the manual.  You can include a URL to the online version of the manual:

http://mc-stan.org/users/documentation